### DISCUSSION TITLE
I made the Scratch Addons logo in Minecraft :)

### DISCUSSION THREAD

**Discussion by Secret-chest** [ScratchAddons/ScratchAddons#1696] [Show and tell]
![image](https://user-images.githubusercontent.com/74449186/109505099-ed561900-7aa4-11eb-8f9f-b34281e471b3.png)

**Comment by ghost** [comment]
Good

**Comment by Endermaster3690** [comment]
Really cool! Vertical slabs not in minecraft really messed it up. Cool though!

Edit: never mind thats the logo

**Comment by Mr-Programmer-777** [comment]
Pretty cool!

**Comment by xItzDiamondx** [comment]
Nice
